src/document/DocumentManager.ts

```diff
@@ -205,6 +205,13 @@ export class DocumentManager {
     hotkeys("command+n,ctrl+n", { keydown: true }, () => {
       this.newFile();
     });
+    hotkeys("command+f,ctrl+f", () => {
```

Comment:
Isn't `Ctrl + F` typically search?

Reply:
Ctrl + 0 is common for default zoom. Is "zoom to fit waypoints" effectively our default?

Reply:
Yes!